extensions/pestphp-documentation/src/pestphp.tsx

```diff
@@ -0,0 +1,163 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  getPreferenceValues,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+} from "@raycast/api";
+import { useEffect, useMemo, useState } from "react";
+import algoliaSearch from "algoliasearch/lite";
+import striptags from "striptags";
+
+type docList = {
+  [versions: string]: {
+    title: string;
+    url: string;
+  }[];
+};
+
+const documentation: { [key: string]: docList } = {
+  "v1.x": require("./documentation/v1.x.json"),
+};
+
+const APPID = "BH4D9OD16A";
+const APIKEY = "d25b0b281fd72e7b8ce770857fff45bf";
```

Comment:
Is this alright to share the API key? I'm curious if there are any limits per key that end-users can easily reach.
Probably worth having it as preferences so anyone interested can configure their own tokens?

Reply:
i believe this is Search-only API key provided by algolia safe to use it on client side, its per site and only data owner can generate them. however pestphp documentation site are using same APIKEY.